pkg/controllers/disruption/consolidation_test.go

```diff
@@ -238,6 +242,9 @@ var _ = Describe("Consolidation", func() {
 			ExpectReconcileSucceeded(ctx, disruptionController, client.ObjectKey{})
 			wg.Wait()
 
+			// Process the item so that the nodes can be deleted.
+			ExpectReconcileSucceeded(ctx, queue, types.NamespacedName{})
```

Comment:
Why are we passing an empty namespaced name in?

Reply:
The queue reconciliation just pops off an item, rather than relying on a source. This is the same as the termination queue as well. We modeled it as a reconciliation because the result handling was effectively the same as controller reconciles - Do we requeue? How long should we requeue for? Was there an error? 

The only difference is that we don't actually return an error now, we just log it and don't requeue the item, so there could be a reason to depart from this semantic. What's your thought?

Reply:
> What's your thought?

Would love to just model this as a standard controller.